docs/developer_guide/secrets.md

```diff
@@ -0,0 +1,132 @@
+# Handling project secrets
+
+Arnold uses OpenShift's `Secret` objects to securely inject services credentials
+to running pods. Those `Secret`s are generated from a YAML file stored in:
+
+```
+group_vars/secret/{{ customer }}/{{ env_type }}/{{ application }}/credentials.vault.yml
+```
+
+Credentials files are securely versioned in Arnold's repository thanks to the
+`ansible-vault` tool (see usage in the next sections). In plain text, the
```

Comment:
Credentials files are securely encrypted using the `ansible-vault` tool before being versioned in Arnold's repository.